stdlib/public/core/Range.swift

```diff
@@ -147,6 +147,29 @@ public struct Range<
   }
 }
 
+/// O(1) implementation of `contains()` for ranges of comparable elements.
+extension Range where Element: Comparable {
+  @warn_unused_result
+  func _customContainsEquatableElement(element: Generator.Element) -> Bool? {
+    return element >= self.startIndex && element < self.endIndex
+  }
+    
+  // FIXME: copied from SequenceAlgorithms as a workaround for https://bugs.swift.org/browse/SR-435
+  @warn_unused_result
+  public func contains(element: Generator.Element) -> Bool {
+    if let result = _customContainsEquatableElement(element) {
```

Comment:
This condition can never fail, so you may as well replace the whole body of this method with the same `return element >= self.startIndex && element < self.endIndex` that you use in `_customContainsEquatableElement`.